src/analytics/events.ts

```diff
@@ -0,0 +1,196 @@
+
+export const EthVMEvent = {
+  ETHVM_LINK_CLICKED: 'EthVMLinkClicked',
+} as const
+export type EthVMEvent = typeof EthVMEvent[keyof typeof EthVMEvent]
+
+export const WalletDirectLinkAccessEvent = {
+  WALLET_DIRECT_LINK_ACCESS: 'WalletDirectLinkAccess'
+} as const
+export type WalletDirectLinkAccessEvent = typeof WalletDirectLinkAccessEvent[keyof typeof WalletDirectLinkAccessEvent]
+
+export const ConsentEvent = {
+  USER_OPT_OUT_TRACKING: 'UserOptOutTracking',
+  USER_OPT_IN_TRACKING: 'UserOptInTracking',
+} as const
+export type ConsentEvent = typeof ConsentEvent[keyof typeof ConsentEvent]
+
+export const LandingPageEvent = {
```

Comment:
hm i think we can all together drop this, as the landing page will not live in this projetc anymore